main/main.go

```diff
@@ -96,7 +92,17 @@ func main() {
 		panic(err)
 	}
 
-	network, responseChans, err := peers.NewNetwork(logger, registerer, cfg.NetworkID, sourceSubnetIDs, sourceBlockchainIDs, cfg.PChainAPIURL)
+	// Initialize the global app request network
+	logger.Info("Initializing app request network")
+	sourceSubnetIDs, sourceBlockchainIDs := cfg.GetSourceIDs()
+
+	// The app request network generates P2P networking logs that are verbose at the info level.
+	// Unless the log level is debug, set the network log level to error to avoid spamming the logs.
+	networkLogLevel := logging.Error
+	if logLevel == logging.Debug {
```

Comment:
The `Verbo` log level is even lower than Debug, should the comparison be `<=`? Also how about log level `Trace`?

Reply:
Good call, fixed